include/trackcpp/passmethods.hpp

```diff
@@ -168,9 +180,19 @@ void bndthinkick(Pos<T>& pos, const double& length,
     T  py = pos.py * pnorm;
     T b2p = b2_perp(imag_sum, real_sum + irho, rx, px, ry, py, irho);
     double radiation_constant =
-      CGAMMA*POW3(accelerator.energy/1e9)/(TWOPI); /*[m]/[GeV^3] M.Sands(4.1)*/
+      CGAMMA*POW3(accelerator.energy/1e9)/(TWOPI); /*[m] M.Sands(4.1)*/
     pos.de -=
       radiation_constant*SQR(1+pos.de)*b2p*(1+irho*rx + (px*px+py*py)/2)*length;
+
+    if (d_factor != 0) {
+      // quantum excitation kick
+      T dl_ds = (1 + rx*irho);
```

Comment:
are you sure about this? did you look in literature to find out about the interpretation of this term?

Reply:
I figured out that the right implementation is $dl/ds = (1+ r_x/\rho + (p_x^2 + p_y^2)/2)$, as @xresende did for the radiation damping case and the _elegant_  uses this expression too. Its also necessary to include a term $*(1+\delta)^2$ to make the most energetic electrons emit more radiation.

I fixed both errors, thanks!